src/routes.js

```diff
@@ -1,36 +1,64 @@
 const express = require("express");
 const multer = require("multer");
 
-const JeController = require('./controllers/JeController');
-const MemberController = require('./controllers/MemberController');
-const DutyController = require('./controllers/DutyController');
-//const authMiddleware = require('./middlewares//auth');
-const multerMiddleware = require('./middlewares/multer');
-const LoginController = require('./controllers/LoginController');
-const authJe = require('./middlewares/authJe');
-const authMember = require('./middlewares/authMember');
+const FeedbackController = require("./controllers/FeedbackController");
+const JeController = require("./controllers/JeController");
+const MemberController = require("./controllers/MemberController");
+const DutyController = require("./controllers/DutyController");
+const multerMiddleware = require("./middlewares/multer");
+const LoginController = require("./controllers/LoginController");
+const authJe = require("./middlewares/authJe");
+const authMember = require("./middlewares/authMember");
```

Comment:
Se puder deixar mais organizado os imports aí
Deixar o q for middleware junto, deixar os controllers juntos e etc etc etc